### ISSUE TITLE
Section without final newline merges to next section in narrow editing

### ISSUE THREAD

**Issue by dalbandea** [amake/orgro#125]
Orgro version: 1.49.4 (from F-Droid)
Android version: 14

I normally edit files in narrow mode (with autosave), and if I don't leave a blank line at the end of the section it will merge the last line with the header of the next section. If file looks like this:
```
* Section 1
Hi
* Section 2
Hi
```
and I want to edit Section 1 by narrowing, to keep section integrity I need to leave an empty line at the end:
```
* Section 1
Hi
(Empty line)
```
Otherwise, if I remove empty line
```
* Section 1
Hi
```
And get out of the file and open it again, the file becomes
```
* Section 1
Hi* Section 2
Hi
```

**Comment by amake** [comment]
Thanks for the report. I think the correct thing to do is simply automatically append a newline to the edited content if there isn't one already. Does that work for you?

**Comment by dalbandea** [comment]
That seems like a good solution, yes. Does the alpp give you this option already? (I don't find a Settings page in the app)

**Comment by amake** [comment]
This will be fixed in v1.50.2, ready for testing soon:

- [iOS](https://testflight.apple.com/join/8YTxcuZC)
- [Android](https://play.google.com/store/apps/details?id=com.madlonkay.orgro)

**Comment by amake** [comment]
This is now released everywhere.
